<?php

try {
    throw new RuntimeException();
} catch (RuntimeException $foo) {

}
